package.json

```diff
@@ -11,17 +11,23 @@
   "devDependencies": {
     "@types/express": "^4.17.13",
     "@types/node": "^16.11.6",
+    "@typescript-eslint/eslint-plugin": "^5.3.1",
+    "@typescript-eslint/parser": "^5.3.1",
+    "eslint": "^8.2.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-plugin-import": "^2.25.3",
     "nodemon": "^2.0.14",
-    "typescript": "^4.4.4",
     "prettier": "^2.4.1",
-    "prettier-airbnb-config": "^1.0.0"
+    "prettier-airbnb-config": "^1.0.0",
+    "typescript": "^4.4.4"
   },
   "scripts": {
     "build": "tsc",
     "start": "tsc & node .",
     "dev": "tsc -w & nodemon .",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "format": "prettier -c src/*"
+    "format": "prettier -c src/*",
+    "lint": "npx eslint src/*"
```

Comment:
This shouldn't need the npx prefix. That's for when you need run scripts in ./node_modules/bin from the command line.

Reply:
resolved